about.json

```
@@ -3,6 +3,10 @@
   "about_url": null,
   "license_url": null,
   "assets": {
+    "archivo": "assets/Archivo-Regular.ttf",
+    "archivo-bold": "assets/Archivo-Bold.ttf",
+    "merriweather": "assets/Merriweather-Regular.ttf",
```

Two of those didn't match the vars in the scss file:

```suggestion
    "archivo-regular": "assets/Archivo-Regular.ttf",
    "archivo-bold": "assets/Archivo-Bold.ttf",
    "merriweather-regular": "assets/Merriweather-Regular.ttf",
```